app/router/fees/fees.go

```diff
@@ -48,3 +52,40 @@ func feesNftResponse(pricingService service.Pricing) func(w http.ResponseWriter,
 		render.JSON(w, r, res)
 	}
 }
+
+func calculateForResponse(feeService service.Fee, feePolicyHandler service.FeePolicyHandler) func(w http.ResponseWriter, r *http.Request) {
+	return func(w http.ResponseWriter, r *http.Request) {
+		targetChainIdStr := r.URL.Query().Get("chain")
```

Comment:
I think targetChain would be a more descriptive parameter name here. We are setting the variable to this why not the parameter as well